Facebook.Android/source/audience-network-sdk/audience-network-sdk.csproj

```diff
@@ -55,7 +55,7 @@
 
   <ItemGroup>
 
-    <PackageReference Include="Xamarin.GooglePlayServices.Basement" Version="71.1620.0" PrivateAssets="None" />
+    <PackageReference Include="Xamarin.GooglePlayServices.Basement" Version="117.6.0.6" PrivateAssets="None" />
```

Comment:
Is this the most recent version, or the version needed by facebook sdk 13?

Reply:
Same question applies to other nuget updates.

Reply:
The changes were necessary to support Android 12. 
these are the most recent versions, not the lowest needed.

Reply:
I suggest to go for the lowest needed to avoid enforcing users to go for the newest version through a transitive dependency.